backend/src/core/services/s3-client.class.ts

```diff
@@ -37,9 +38,7 @@ export default class S3Client {
    * Deduplicates the csv by overriding the same recipient with newer records
    * @param readStream
    */
-  async parseCsv(
-    readStream: NodeJS.ReadableStream
-  ): Promise<Array<CSVParamsInterface>> {
+  async parseCsv(readStream: NodeJS.ReadableStream): Promise<Array<CSVParams>> {
```

Comment:
Would it be easier if we move `parseCsv` out of `S3Client` at this point so that it will be easier when we do get around to refactoring it?

Reply:
That's a good idea. Will put it inside core services.